lib/core/rollup.config.js

```diff
@@ -0,0 +1,280 @@
+import fs from "fs";
+import path from "path";
+import vue from "rollup-plugin-vue";
+import alias from "@rollup/plugin-alias";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
```

Comment:
I really think you should read this config and simplify it much more :D the "commonjs", for example, you should not need, because we won't bundle 3rd party libraries inside our lib. This plugin is only used for "app", not "lib"